[2024-12-27 07:02:35,423: INFO: main: Welcome to our custom log]
[2024-12-28 21:23:30,296: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-28 21:25:47,016: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-28 21:26:21,282: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-28 21:27:11,656: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-28 21:29:12,469: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-28 21:31:27,244: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-28 21:35:10,217: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-28 21:40:26,904: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-28 21:41:35,531: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-29 09:10:37,344: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-29 09:45:33,471: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-29 09:45:33,472: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-29 09:45:33,472: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-29 09:45:33,473: INFO: common: Created directory at: artifacts]
[2024-12-29 09:45:33,473: INFO: common: Created directory at: artifacts/data_ingestion]
[2024-12-29 09:45:33,473: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-12-29 09:45:44,291: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-12-29 09:45:44,669: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-29 09:45:44,669: INFO: main: *******************]
[2024-12-29 09:45:44,669: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-12-29 09:45:44,670: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-12-29 09:45:44,671: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-29 09:45:44,671: INFO: common: Created directory at: artifacts]
[2024-12-29 09:45:44,671: ERROR: main: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config']
Traceback (most recent call last):
  File "main.py", line 29, in <module>
    prepare_base_model.main()
  File "/Users/shafins/Documents/Learning/M.Tech AI and ML/Sem 3/trialprojects/Kidney-Disease-Classification-Deep-Learning-Project/src/cnnClassifier/pipeline/stage_02_prepare_base_model.py", line 15, in main
    prepare_base_model_config = config.get_prepare_base_model_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config'
